scripts/monorepo.sh

```diff
@@ -7,8 +7,10 @@ BRANCH=$3
 # This script is highly specific to circleci and will "succeed" early when the
 # current git changes are not relevant to the folder being tested (and not on master)
 
+SHARED_CONFIG_FILES=(package.json .eslintrc.js .prettierrc)
```

Comment:
This is really the only important change here: it makes sure that if we change any of these 3 files, tests (including linting) will be run again in the subdirectory.

This should reduce the likeliness of this mess.